### PR TITLE
Add decline reason for work history

### PR DESCRIPTION
**Pull request by David-Feetenby**
Add  new decline reason for insufficient work history duration

### CODE REVIEW

File: app/lib/failure_reasons.rb [hunk 1]
```diff
@@ -29,6 +29,8 @@ class FailureReasons
       "teaching_qualifications_from_ineligible_country",
     TEACHING_QUALIFICATIONS_NOT_AT_REQUIRED_LEVEL =
       "teaching_qualifications_not_at_required_level",
+    TEACHING_QUALIFICATIONS_COMPLETED_IN_ELIGIBLE_COUNTRY =
```
Comment (thomasleese) [inline app/lib/failure_reasons.rb:32]: I'm not sure that we need this line, it'll mean that we end up defining `TEACHING_QUALIFICATIONS_COMPLETED_IN_ELIGIBLE_COUNTRY` as `work_history_duration`.

**Review by thomasleese — COMMENTED** [review]
Just a minor comment, otherwise it looks good!

File: app/lib/failure_reasons.rb (hunk 1 above)
Comment (David-Feetenby) [reply app/lib/failure_reasons.rb:32]: Not sure why i left that in 😆

File: app/lib/failure_reasons.rb [hunk 2]
```diff
@@ -29,6 +29,8 @@ class FailureReasons
       "teaching_qualifications_from_ineligible_country",
     TEACHING_QUALIFICATIONS_NOT_AT_REQUIRED_LEVEL =
       "teaching_qualifications_not_at_required_level",
+    WORK_HISTORY_DURATION =
+      WORK_HISTORY_DURATION = "work_history_duration",
```
Comment (thomasleese) [inline app/lib/failure_reasons.rb:33]:
```suggestion
    WORK_HISTORY_DURATION =
      "work_history_duration",
```

The variable only needs to defined once :)

**Review by thomasleese — APPROVED** [review]

**Pull request merged**